lib/utils.pm

```diff
@@ -1216,7 +1216,8 @@ sub set_hostname {
 
                 # poo#169726 Increasing timeout to 120s and adding DEBUG logs for future investigation
                 script_run("nmcli general logging level DEBUG");
-                assert_script_run("nmcli -w 120 device disconnect $dev");
+                # poo#184165 By default sle16 qcow created in openqa will not bring up all interface automaticly
```

Comment:
Mentioned poo makes impression, that there is something pending. If it is valid solution for SLE16, i would change comment to `Don't disconnect devices on SLE16, they will not be bring up automatically`. By the way why test needed to disconnect devices on SLE15?

Reply:
SLE15 use wicked instead of NetworkManager so it will not do this action.